docs/source/conf.py

```diff
@@ -1751,6 +1751,8 @@
     "verify_aten_graph",
     # torch.optim.adadelta
     "adadelta",
+    # torch.optim.adafactor
+    "adafactor",
```

Comment:
No new entry should be added to this list. Please make sure they are properly documented on the website